### PR TITLE
automate issue linked label

### PR DESCRIPTION
**Pull request by dan-mba**
## Changes proposed
Add code to automate `issue linked ` label to the labeler action

See this repo for my testing:
https://github.com/dan-mba/label-test

This will only work if the issue is mentioned correctly so that GitHub displays it here:
Screen Shot 2023-08-09 at 07 30 47

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8581"></a>

### CODE REVIEW

File: .github/workflows/labels.yml [hunk 1]
```diff
@@ -23,3 +24,44 @@ jobs:
...
+  issue-linked:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: octokit/graphql-action@v2.x
+        name: Get PR data from GraphQL API
+        id: get_linked_issue
+        with:
+          query: |
+            query issue($owner:String!,$repo:String!,$pr:Int!) {
```
Comment (eddiejaoude) [inline .github/workflows/labels.yml:36]: Love these improvements!!

When we manually add the label, we also check the author of the PR is who the issue is assigned to also, is that already done or is that something we can do?

Comment (dan-mba) [reply .github/workflows/labels.yml:36]: I will look int it

Comment (dan-mba) [reply .github/workflows/labels.yml:35]: @eddiejaoude The updated code handles the assignee check

**Comment by eddiejaoude** [comment]
Wow this is epic Dan!! 🔥

**Review by eddiejaoude — APPROVED** [review]
Awesome!

**Pull request merged**